### PR TITLE
Tidy up scripts & improve help message

### PR DESCRIPTION
**Pull request by AlanGriffiths**

### CODE REVIEW

File: bin/wayland-launch [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-  echo "Wayland interface not connected!"
-  exit 1
-fi
+for PLUG in %PLUGS%; do
+  if ! snapctl is-connected ${PLUG}
+  then
+    echo "${PLUG} interface not connected! Please run: /snap/%SNAP%/current/bin/setup.sh"
+    exit 1
+  fi
+done
```
Comment (Saviq) [inline bin/wayland-launch:11]: This seems excessive, at least some of the interfaces can be optional, no?

**Review by Saviq — CHANGES_REQUESTED** [review]

File: bin/wayland-launch (hunk 1 above)
Comment (AlanGriffiths) [reply bin/wayland-launch:11]: Hmm, I was thinking the interfaces connected by the `setup.sh` script would be required. (There can obviously be other interfaces not mentioned to `build-with-plugs.sh`.)

Maybe there should be two build scripts? One for "setup" and one for "launch"?

Or at least, better documentation of the presumptions.

Thoughts?

Comment (Saviq) [reply bin/wayland-launch:11]: Two scripts sounds excessive, I'd be OK with this if we said `%PLUGS%` _are_ indeed required.

**Review by Saviq — APPROVED** [review]
A'ight.

**Pull request merged**